src/libstd/io/stdio.rs

```diff
@@ -872,6 +874,40 @@ impl fmt::Debug for StderrLock<'_> {
     }
 }
 
+/// Prints the given `str` and reads a [`String`] from
+/// [standard input](Stdin). The trailing newline is stripped.
+/// Gives an error on EOF (Ctrl-D).
```

Comment:
This is Ctrl+Z on windows. Maybe it's best just not to mention the exact key?
```suggestion
/// Gives an error on end of file.
```

Reply:
Haha, I did not put ctrl-z since I don't use windows.